rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/api/BakeRequest.groovy

```diff
@@ -56,14 +56,15 @@ class BakeRequest {
   String ami_name
   String ami_suffix
   Boolean upgrade
+  String instance_type
```

Comment:
So at some point, you'll want to add this same `instance_type` attribute to orca's `BakeRequest` here: https://github.com/spinnaker/orca/blob/master/orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/api/BakeRequest.groovy#L30

That will allow orca to unmarshall it in requests coming from the ui (deck). Then, you'll need to ensure that it only gets sent to Rosco, and is not included on bake requests propagated from orca to the Netflix internal bakery: https://github.com/spinnaker/orca/blob/master/orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/tasks/CreateBakeTask.groovy#L146